src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categories = createSlice({
+  name: 'category',
+  initialState: { category: [] },
+  reducers: {
+    checkStatus() {
+      return 'Under construction';
+    },
+  },
+});
+
+export const categoriesActions = categories.actions;
+
+export default categories;
```

Comment:
- [ ] The same thing here, Please check the comment above ⬆

Reply:
Hi @Bondok6 
I think you are missing the point of ducks pattern, the requirement is that 
- It must export a reducer function,
- export an action creator
- MUST have action types in the form npm-module-or-app/reducer/ACTION_TYPE (which redux toolkit is an npm module)
All this has been followed hence the ducks pattern is followed.
You can also reference the [redux official documentation](https://redux.js.org/style-guide/) 
Which also recommends using redux toolkit and states clearly that it follows the ducks pattern. 
(You can just search for ducks pattern on the page to easily find the reference).